api/src/test/java/org/pmiops/workbench/institution/InstitutionServiceTest.java

```diff
@@ -956,6 +956,53 @@ public void testERARequired() {
         .isEqualTo(true);
   }
 
+  @Test
```

Comment:
please add tests for:
* user is eligible for CT
* user's institution has tier requirements for RT and CT, but the user is only eligible for CT